frontend/src/components/ui/AppAlertBanner.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <!--  -->
```

Comment:
Please fix my mistake and remove this :)

Reply:
removed